tests/integration/test_tasks.py

```diff
@@ -42,3 +42,20 @@ def test_list_tasks(client: TestClient, user_access_token: str, internal_study_i
 
     assert len(res_task_list) == 2
     assert actual_task_list == expected_task_list
+
+    # pour les status, faire des GET récupérant les COMPLETED et les RUNNING (confirmer qu'il n'y en a pas)
```

Comment:
You can remove this comment. We dont want french in the code in general, and I dont think this comment is relevant